<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Saif" name="com.acorns.testcases.HomePageTest" tests="7" failures="1" timestamp="07 Sep 2018 03:34:51 GMT" time="160.695" errors="0">
  <testcase name="clickOnInvesmentLinkTest" time="39.497" classname="com.acorns.testcases.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [Learn More About Our Portfolios | corns] but found [Learn More About Our Portfolios | Acorns]">
      <![CDATA[java.lang.AssertionError: expected [Learn More About Our Portfolios | corns] but found [Learn More About Our Portfolios | Acorns]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.acorns.testcases.HomePageTest.clickOnInvesmentLinkTest(HomePageTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- clickOnInvesmentLinkTest -->
  <testcase name="clickOnFoundMoneyLinkTest" time="35.423" classname="com.acorns.testcases.HomePageTest"/>
  <testcase name="clickOnAcornsLaterLinkTest" time="18.399" classname="com.acorns.testcases.HomePageTest"/>
  <testcase name="clickOnSpendLinkTest" time="20.021" classname="com.acorns.testcases.HomePageTest"/>
  <testcase name="clickOnPricingLinkTest" time="15.228" classname="com.acorns.testcases.HomePageTest"/>
  <testcase name="clickOnGrowMagazineLinkTest" time="12.633" classname="com.acorns.testcases.HomePageTest"/>
  <testcase name="clickOnGiftCArdLinkTest" time="19.494" classname="com.acorns.testcases.HomePageTest"/>
</testsuite> <!-- com.acorns.testcases.HomePageTest -->
